packages/core/upload/admin/src/hooks/tests/useConfig.test.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { QueryClientProvider, QueryClient, useQueryClient } from 'react-query';
+import { renderHook, act } from '@testing-library/react-hooks';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+
+import { NotificationsProvider, useNotification } from '@strapi/helper-plugin';
+
+import { sortOptions, pageSizes } from '../../constants';
+import { axiosInstance } from '../../utils';
+import { useConfig } from '../useConfig';
+import pluginId from '../../pluginId';
+
+const getResponse = {
+  data: {
+    data: {
+      pageSize: pageSizes[0],
+      sort: sortOptions[0].value,
+    },
+  },
+};
+
+jest.mock('../../utils', () => ({
+  ...jest.requireActual('../../utils'),
+  axiosInstance: {
+    put: jest.fn(() => {
+      const res = { data: { data: {} } };
+
+      return Promise.resolve(res);
+    }),
+    get: jest.fn(() => getResponse),
```

Comment:
I'm wondering why this works 🤔 Doesn't `get` return a Promise?

```suggestion
    get: jest.fn().mockResolvedValue(getResponse),
```